deployment/hasura/metadata/databases/AerieScheduler/tables/public_scheduling_specification.yaml

```diff
@@ -34,7 +34,7 @@ update_permissions:
       filter: {}
   - role: user
     permission:
-      columns: [horizon_start, horizon_end, simulation_arguments, analysis_only]
+      columns: [plan_revision, horizon_start, horizon_end, simulation_arguments, analysis_only]
```

Comment:
`aerie_admin` also needs this change.

Reply:
I believe `aerie_admin` already has update permissions on this column. In practice, scheduling/analysis does work while using admin role.

Reply:
Ah, you're right, misread it. 🚀